samples/endpoints/host/src/register.tsx

```diff
@@ -34,15 +41,15 @@ function registerRoutes(runtime: FireflyRuntime) {
     });
 
     runtime.registerNavigationItem({
-        $label: "Home",
+        $label: <I18nextNavigationLabel i18next={i18nextInstance} resourceKey="homePage" />,
```

Comment:
Is the Label suffix possibly misleading for this component? it returns a string, not a <label>.

Reply:
The reason I suffixed with `Label` is becaused this component is expected to strictly be used for the Label field of a NavigationItem.

I renamed the component to `I18nextNavigationItemLabel` by the way.